### PR TITLE
Feat i18n apply

### PR DESCRIPTION
**Pull request by mixcas**
Add translated i18n labels to all fields in apply.

I has an issue, it won't refresh the component on language switch. I think it stopped working after ec0666c. I'm gonna look into it but you can start reviewing this code.

### CODE REVIEW

**Comment by mixcas** [comment]
Ok, fixing the lang switch would be done in a diff branch since it requires more work than i thought.

This PR is ready.

File: app/imports/components/fields/signatureField.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* globals SignaturePad */
 import React, { Component } from 'react';
 
-import { connectField } from 'uniforms';
+import { connectField, filterDOMProps } from 'uniforms';
```
Comment (varffs) [inline app/imports/components/fields/signatureField.jsx]: do you actually use `filterDOMProps`?

Comment (mixcas) [inline app/imports/components/fields/signatureField.jsx]: oh, not anymore :O
should remove this line

File: app/imports/schemas/galleryInformation.js [hunk 2]
```diff
@@ -1,289 +1,306 @@
...
+          {value: "Vanuatu", label: "Vanuatu",},
+          {value: "Venezuela, Bolivarian Republic of", label: "Venezuela, Bolivarian Republic of",},
+          {value: "Viet Nam", label: "Viet Nam",},
+          {value: "Virgin Islands, British", label: "Virgin Islands, British",},
+          {value: "Virgin Islands, U.S.", label: "Virgin Islands, U.S.",},
+          {value: "Wallis and Futuna", label: "Wallis and Futuna",},
+          {value: "Western Sahara", label: "Western Sahara",},
+          {value: "Yemen", label: "Yemen",},
+          {value: "Zambia", label: "Zambia",},
+          {value: "Zimbabwe", label: "Zimbabwe",},
```
Comment (varffs) [inline app/imports/schemas/galleryInformation.js:288]: these aren't localized but dont think that really matters?

Comment (mixcas) [inline app/imports/schemas/galleryInformation.js:288]: thats what i thought

**Comment by varffs** [comment]
This looks good. I wonder if some of the values should be loaded from settings or if that is pointless? Not in this PR anyhow

**Comment by mixcas** [comment]
@varffs like what?

**Comment by varffs** [comment]
All of it!?

**Comment by mixcas** [comment]
cmon be specific

**Comment by varffs** [comment]
All of it is specific. All of the localized titles etc. But I guess it makes no sense as the point of applaya is to customize each time. And unless the whole thing was creatable with config why make dependent parts

**Comment by mixcas** [comment]
isnt that what those translation file are? they have to be customized for each project

**Comment by varffs** [comment]
Yeah basically whats what im saying now. Thinking aloud too tired so pointless idea

**Comment by varffs** [comment]
There are some mega lint errors but might be incompatibility between lint config and the format required for i18n package? anyhow im merging

**Pull request merged**